client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -61,7 +62,8 @@ class ProductCreate extends React.Component {
 	}
 
 	onSave = () => {
-		// TODO: Add action dispatch to save this product.
+		const { siteId, product } = this.props;
```

Comment:
Yeah, I'll definitely be looking to change the way siteId is tracked for the edit state.